Submit to F-Droid

### Feature description

It would be great to get this app through [F-Droid](https://f-droid.org/).

### Why do you want this feature?

F-Droid is *the* app store for open source apps.

### Additional information

- [Inclusion Policy](https://f-droid.org/en/docs/Inclusion_Policy/)
- [How to submit](https://f-droid.org/en/contribute/#submit-applications)

I can provide some helping hands, if you are ok with F-Droid.

@vfsfitvnm afaik non-final Gradle versions are not FOSS, in my recipe we downgrade to 7.4

But that's no solution:
```
FAILURE: Build failed with an exception.

* Where:
Build file '/home/vagrant/build/it.vfsfitvnm.vimusic/app/build.gradle.kts' line: 1

* What went wrong:
An exception occurred applying plugin request [id: 'com.android.application']
> Failed to apply plugin 'com.android.internal.version-check'.
   > Minimum supported Gradle version is 7.5-rc-2. Current version is 7.4. If using the gradle wrapper, try editing the distributionUrl in /home/vagrant/build/it.vfsfitvnm.vimusic/gradle/wrapper/gradle-wrapper.properties to gradle-7.5-rc-2-all.zip
```

--- a/README.md
+++ b/README.md
@@ -34,7 +34,13 @@
 - ...
 
 ## Installation
-[<img src="https://github.com/machiav3lli/oandbackupx/blob/034b226cea5c1b30eb4f6a6f313e4dadcbb0ece4/badge_github.png" alt="Get it on GitHub" height="80">](https://github.com/vfsfitvnm/ViMusic/releases/latest)
+
+[<img src="https://fdroid.gitlab.io/artwork/badge/get-it-on.png"
+     alt="Get it on F-Droid"
+     height="80">](https://f-droid.org/packages/it.vfsfitvnm.vimusic/)
+[<img src="https://github.com/machiav3lli/oandbackupx/blob/034b226cea5c1b30eb4f6a6f313e4dadcbb0ece4/badge_github.png"
+      alt="Get it on GitHub"
+      height="80">](https://github.com/vfsfitvnm/ViMusic/releases/latest)
 
 After installing, I recommend executing the following ADB command to neutralize some animation lags you may experience in cold starts:
 ```

